app/views/projects/_content_item.html.erb

```diff
@@ -16,6 +16,11 @@
     </h4>
 
     <p><%= content_item.description %></p>
+
+    <% if content_item.needs_help? && content_item.need_help_comment.present? %>
+      <p>Comment: <%= content_item.need_help_comment %></p>
```

Comment:
Perhaps this text could shown next to the `help-needed` flag, similar to the "Suggested topic"?